src/transport/SessionManager.cpp

```diff
@@ -736,27 +755,75 @@ void SessionManager::SecureUnicastMessageDispatch(const PacketHeader & packetHea
     }
 }
 
-void SessionManager::SecureGroupMessageDispatch(const PacketHeader & packetHeader, const Transport::PeerAddress & peerAddress,
+/**
+ * Helper function to implement a single attempt to decrypt a groupcast message
+ * using the given group key and privacy setting.
+ *
+ * @param[in] packetHeader The partial packet header of the non-obfuscated fields of the message header
```

Comment:
I think it's worth documenting what "partial" means here: It's had DecodeFixed called on it.